tasks/managed/extract-oot-kmods/extract-oot-kmods.yaml

```diff
@@ -114,26 +114,24 @@ spec:
         KMODS_PATH=$(params.kmodsPath)
         SIGNED_KMODS_PATH=$(params.signedKmodsPath)
         snapshot_json=$(cat "$(params.dataDir)/$(params.snapshotPath)")
-        sha_image=$(jq -r '.metadata.annotations["build.appstudio.redhat.com/commit_sha"]' <<< "$snapshot_json")
-        source_image=$(jq -r '.spec.components[].containerImage' <<< "$snapshot_json" | sed 's/@.*$//')
-        jq -r '.spec.components[].containerImage' <<< "$snapshot_json"
-        SRC_IMAGE="$source_image:$sha_image"
-        TMP_DIR=$(mktemp -d)
-        echo "Copying $SRC_IMAGE to $TMP_DIR..."
-        skopeo copy docker://"$SRC_IMAGE" dir:"$TMP_DIR"
+        source_image=$(jq -r '.components[].containerImage' <<< "$snapshot_json")
```

Comment:
Is this only supposed to work if there is one component? Otherwise `.components[].containerImage` would return each component's `containerImage`, no?

Reply:
The intended behavior is for a single component but maybe I can add some check to validate there's just one component.